deploy/render/__init__.py

```diff
@@ -1 +1,10 @@
 from viur.core.render import admin, html, json, vi, xml
+
+@html.utils.jinjaGlobalFilter
+def isList(render, val):
+    return isinstance(val, list)
+
+
+@html.utils.jinjaGlobalFilter
+def isDict(render, val):
+    return isinstance(val, dict)
```

Comment:
Please don't re-add old stuff

```suggestion

@html.utils.jinjaGlobalFilter
def isList(render, val):
    return isinstance(val, list)


@html.utils.jinjaGlobalFilter
def isDict(render, val):
    return isinstance(val, dict)
```